sdk/tools/isohybrid/CMakeLists.txt

```diff
@@ -5,3 +5,11 @@ add_definitions(
 add_host_tool(isohybrid
     isohybrid.c
     reactos_support_code.c)
+
+if(MSVC)
+    # Disable warning "'x': unreferenced local variable"
+    add_target_compile_flags(isohybrid "/wd4101")
+
+    # Disable warning "'return': conversion from '__int64' to 'int', possible loss of data"
+    add_target_compile_flags(isohybrid "/wd4244")
+endif()
```

Comment:
Not sure I understand your idea behind these changes. You're fixing some warnings properly and others are just disabled. Would a proper fix take too much time?

These changes should later be submitted upstream either way, but as my last one-liner patch took 4 months to be merged (https://repo.or.cz/syslinux.git/commit/80b0ef5ffd6998818b3785c3c15bf2ae73687e09), a single patch should fix as much as possible :)

Reply:
I ignored most 3rd-party code, since I prefer to not waste my time with trying to get all these tiny things upstream, especially considering that these are MSVC specific and we that we use a pretty strict set of warnings that others could easily consider too picky. You are welcome though to fix those properly and get a patch upstream :)

Reply:
Fixed those with my commit to your PR